torch/testing/_internal/opinfo/definitions/linalg.py

```diff
@@ -1450,6 +1450,12 @@ def make_input():
                 device_type="mps",
                 dtypes=[torch.float32],
             ),
+            # This appears to be a different algorithm on cpu vs gpu
```

Comment:
I am curious why `eig` is a skip but later on you mark `eigh` xfail?

Reply:
updated the comment, This will fail on CI, but locally passes with a newer cuda toolkit version